runtime/config.go

```diff
@@ -47,11 +47,16 @@ type Config struct {
 	LogLevel              string `json:"log_level"`
 	HtEnabled             bool   `json:"ht_enabled"`
 	Debug                 bool   `json:"debug"`
-
 	// If a CreateVM call specifies no network interfaces and DefaultNetworkInterfaces is non-empty,
 	// the VM will default to using the network interfaces as specified here. This is especially
 	// useful when a CNI-based network interface is provided in DefaultNetworkInterfaces.
 	DefaultNetworkInterfaces []proto.FirecrackerNetworkInterface `json:"default_network_interfaces"`
+	JailerConfig             JailerConfig                        `json:"jailer"`
```

Comment:
Nit: can you rename either the JSON key (to jailer_config) or the member itself (to Jailer), to be consistent with others?

Reply:
The member definitely should not be `Jailer`. The reason I chose `jailer` for the `json` is that I think it makes a lot of sense due to configuration being implied since you are in a configuration file. Having `json` tags be consistent isn't really an issue. However, if others feel the same way I won't mind changing it.

Reply:
I see. I'm fine calling that `jailer` here.